documentation/modules/overview/con-key-features-kafka-connect.adoc

```diff
@@ -0,0 +1,102 @@
+
+Connector plugins provide the implementation for Kafka Connect to run connector instances.
+Connector instances create and orchestrate the tasks required to transfer data in and out of Kafka.
+Tasks split the work required.
+The number of tasks created relates to the number of Kafka topics and partitions.
+
+[NOTE]
+--
+Kafka provides two built-in connectors:
+
+* `FileStreamSourceConnector` streams data from an external system to Kafka, reading lines from an input source and sending each line to a Kafka topic.
+* `FileStreamSinkConnector` streams data from Kafka to an external system, reading messages from a Kafka topic and creating a line for each in an output file.
+--
+
+== Workers
+
+Workers are individual pods in a Kubernetes cluster.
+A Kafka Connect cluster represents a group of workers with the same group id.
+The workers are assigned one or more connector instances and tasks.
+Workers perform processes for the connectors and tasks.
+
+The distributed approach to deploying Kafka Connect is fault tolerant and scalable.
+If a worker node fails, active workers share the tasks.
+You can add to a group of worker nodes through configuration of the `replicas` property in the `KafkaConnect` resource.
+
+Workers store the initial connector configuration deployed to the Kafka Connect cluster.
+They also start connectors and their tasks.
+
+* Source connector workers read events from an external data system and use converters to construct key and value pairs to store in Kafka.
+* Sink connector workers read records from Kafka and use converters to construct records to pass into an external data system.
+
+== Transforms
+
+Kafka Connect translates and transforms external data.
+Single-message transforms change messages into a format suitable for the target destination.
+For example, a transform might insert or rename a field. Transforms can also filter and route data.
+Connector plugins contain the implementation required for workers to perform one or more transformations.
+
+* Source connectors transform data before converting it into a format supported by Kafka.
+* Sink connectors transform data after converting it into a format suitable for an external data system.
+
+Kafka Connect provides a set of standard transforms, but you can also create your own.
+
+== Converters
+
+When a worker receives data, it converts the data into an appropriate format using a converter.
+You specify converters for workers in the worker `config` in the `KafkaConnect` resource.
+
+Kafka Connect can convert data to and from formats supported by Kafka, such as JSON or Avro.
+It also supports schemas for structuring data.
+If you are not converting data into a structured format, you don’t need to enable schemas.
+
+NOTE: You can also specify converters for specific connectors to override the general Kafka Connect worker configuration that applies to all workers.
+
+== Tasks
+
+Data transfer orchestrated by connector instances is split into tasks that run in parallel.
```

Comment:
Again, it's not the connector instance which does orchestration of tasks, it's the connect runtime.